src/assertions.ts

```diff
@@ -366,6 +369,81 @@ export async function runAssertion({
     };
   }
 
+  if (baseType === 'is-sql') {
+    let parsedSQL;
+    let databaseType: string | undefined = 'MySQL';
```

Comment:
Because this has a default, seems like its type may not need to have an `undefined`?

Reply:
True